shell/src/main/java/alluxio/cli/fs/command/StatCommand.java

```diff
@@ -107,16 +126,18 @@ public String getUsage() {
 
   @Override
   public String getDescription() {
-    return "Displays info for the specified path both file and directory."
-        + " Specify -f to display info in given format:"
-        + "   \"%N\": name of the file;"
-        + "   \"%z\": size of file in bytes;"
-        + "   \"%u\": owner;"
-        + "   \"%g\": group name of owner;"
-        + "   \"%y\" or \"%Y\": modification time,"
-        + " %y shows 'yyyy-MM-dd HH:mm:ss' (the UTC date),"
-        + " %Y it shows milliseconds since January 1, 1970 UTC;"
-        + "   \"%b\": Number of blocks allocated for file";
+    return String.join("\n", Arrays.asList(
+        "Displays info for the specified file or directory.",
+        "Specify -file-id to treat the first positional argument as a file ID.",
+        "Specify -f to display info in given format:",
+        "   \"%N\": name of the file;",
+        "   \"%z\": size of file in bytes;",
+        "   \"%u\": owner;",
+        "   \"%g\": group name of owner;",
+        "   \"%y\" or \"%Y\": modification time,",
+        " %y shows 'yyyy-MM-dd HH:mm:ss' (the UTC date),",
+        " %Y it shows milliseconds since January 1, 1970 UTC;",
+        "   \"%b\": Number of blocks allocated for file"));
```

Comment:
Can you please add a new format place holder `%i` that will be substituted with the file ID? This enables a user to get the file ID from its path. If you prefer, you can leave it to another PR.

Reply:
https://github.com/Alluxio/alluxio/issues/15642 I create an issue to track this.